src/ripple/rpc/handlers/WalletPropose.cpp

```diff
@@ -26,13 +27,18 @@ namespace ripple {
 // }
 Json::Value doWalletPropose (RPC::Context& context)
 {
+    return WalletPropose (context.params);
+}
+
+Json::Value WalletPropose (Json::Value const& params)
```

Comment:
Is it not apparent from browsing ripple sources that the convention is to begin function names with a lowercase letter?